smoke/IotEdgeQuickstart/details/IotedgedWindows.cs

```diff
@@ -13,21 +13,23 @@ namespace IotEdgeQuickstart.Details
 
     class IotedgedWindows : IBootstrapper
     {
-        const string ConfigYamlFile = @"C:\ProgramData\iotedge\config.yaml";
+        readonly string configYamlFile = Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData) + @"\iotedge\config.yaml";
 
         readonly string offlineInstallationPath;
         readonly Option<RegistryCredentials> credentials;
         readonly TimeSpan iotEdgeServiceOperationWaitTime = TimeSpan.FromMinutes(5);
         readonly Option<string> proxy;
         readonly Option<UpstreamProtocolType> upstreamProtocol;
+        readonly bool isCleanInstall;
```

Comment:
nit: Suggest to name it "requireEdgeInstallation".

Reply:
Renamed.